canvas_modules/common-canvas/__tests__/common-properties/controls/expression-test.js

```diff
@@ -127,6 +127,10 @@ function getCopy(value) {
 	return JSON.parse(JSON.stringify(value));
 }
 
+function getAddButtonsList(rows) {
+	return rows.find(".ReactVirtualized__Table__rowColumn").find("Button.expression-add-field-button");
```

Comment:
Why do we need `ReactVirtualized__Table__rowColumn`?  I'm asking because this might change in the future.  Wouldn't the button be unique enough?